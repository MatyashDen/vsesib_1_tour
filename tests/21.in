6 12 3 1
5 11
