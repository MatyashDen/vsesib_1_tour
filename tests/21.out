1294
0LRE0UDE0LLRRE0LUDRE0UUDDE0LLLRRRE0LLUDRRE0LUUDDRE0UUUDDDE0LLLLRRRRE0LLLUDRRRE0LLUUDDRRE0LUUUDDDRE0UUUUDDDDE0LLLLLRRRRRE0LLLLUDRRRRE0LLLUUDDRRRE0LLUUUDDDRRE0LUUUUDDDDRE0UUUUUDDDDDE0LLLLLLRRRRRRE0LLLLLUDRRRRRE0LLLLUUDDRRRRE0LLLUUUDDDRRRE0LLUUUUDDDDRRE0LUUUUUDDDDDRE0LLLLLLLRRRRRRRE0LLLLLLUDRRRRRRE0LLLLLUUDDRRRRRE0LLLLUUUDDDRRRRE0LLLUUUUDDDDRRRE0LLUUUUUDDDDDRRE0LLLLLLLLRRRRRRRRE0LLLLLLLUDRRRRRRRE0LLLLLLUUDDRRRRRRE0LLLLLUUUDDDRRRRRE0LLLLUUUUDDDDRRRRE0LLLUUUUUDDDDDRRRE0LLLLLLLLLRRRRRRRRRE0LLLLLLLLUDRRRRRRRRE0LLLLLLLUUDDRRRRRRRE0LLLLLLUUUDDDRRRRRRE0LLLLLUUUUDDDDRRRRRE0LLLLUUUUUDDDDDRRRRE0LLLLLLLLLLRRRRRRRRRRE0LLLLLLLLLUDRRRRRRRRRE0LLLLLLLLUUDDRRRRRRRRE0LLLLLLLUUUDDDRRRRRRRE0LLLLLLUUUUDDDDRRRRRRE0LLLLLUUUUUDDDDDRRRRRE0LLLLLLLLLLLRRRRRRRRRRRE0LLLLLLLLLLUDRRRRRRRRRRE0LLLLLLLLLUUDDRRRRRRRRRE0LLLLLLLLUUUDDDRRRRRRRRE0LLLLLLLUUUUDDDDRRRRRRRE0LLLLLLUUUUUDDDDDRRRRRRE0LLLLLLLLLLLUDRRRRRRRRRRRE0LLLLLLLLLLUUDDRRRRRRRRRRE0LLLLLLLLLUUUDDDRRRRRRRRRE0LLLLLLLLUUUUDDDDRRRRRRRRE0LLLLLLLUUUUUDDDDDRRRRRRRE0LLLLLLLLLLLUUDDRRRRRRRRRRRE0LLLLLLLLLLUUUDDDRRRRRRRRRRE0LLLLLLLLLUUUUDDDDRRRRRRRRRE0LLLLLLLLUUUUUDDDDDRRRRRRRRE0LLLLLLLLLLLUUUDDDRRRRRRRRRRRE0LLLLLLLLLLUUUUDDDDRRRRRRRRRRE0LLLLLLLLLUUUUUDDDDDRRRRRRRRRE0LLLLLLLLLLLUUUUDDDDRRRRRRRRRRRE0LLLLLLLLLLUUUUUDDDDDRRRRRRRRRRE0LLLLLLLLLLLUUUUUDDDDDRRRRRRRRRRRE
WWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
WWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
